source/draw/gpu/opengl/backend/gles3/Gles3ShaderProgram.ooc

```diff
@@ -39,7 +39,7 @@ Gles3ShaderProgram: class extends GLShaderProgram {
 	setUniform: override func ~Int (name: String, value: Int) {
 		version(debugGL) { validateStart("ShaderProgram setUniform~Int") }
 		glUniform1i(glGetUniformLocation(this _backend, name), value)
-		version(debugGL) { validateEnd("ShaderProgram setUniform~Int") }
+		version(debugGL) { validateEnd("ShaderProgram setUniform~Int name:%s value:%d" format(name, value)) }
```

Comment:
Okay, just making sure it wasn't a mistake. So you're planning on adding it to the other functions too, then?